ansible/roles/jitsi-videobridge/tasks/install.yml

```
@@ -131,23 +131,24 @@
     src: "jvb-stats.sh"
     mode: 0755
     owner: root
-  when: jvb_cloud_provider == "aws"
+  when: jvb_cloud_provider == "aws" and jvb_enable_stats_cron
 
 - name: JVB stats script upload oracle
   ansible.builtin.copy:
     dest: "{{ path_to_stats_script }}"
     src: "jvb-stats-oracle.sh"
     mode: 0755
     owner: root
-  when: jvb_cloud_provider == "oracle"
+  when: jvb_cloud_provider == "oracle" and jvb_enable_stats_cron
 
 # add a stats script to upload colibri stats to DataDog
 - name: JVB stats python script upload
   ansible.builtin.copy:
     dest: "{{ path_to_stats_python_script }}"
-    src: "jvb-stats.py"
+    src: "{{ jvb-stats-extras-only.py if jvb_enable_telegraf else jvb-stats.py }}"
```

this probably needs to be quoted?  Did it work without it?

```suggestion
    src: "{{ 'jvb-stats-extras-only.py' if jvb_enable_telegraf else 'jvb-stats.py' }}"
```